### PR TITLE
Feature/prlc100 206

### PR DESCRIPTION
**Pull request by kumarmadurai**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.test.ts [hunk 1]
```diff
@@ -0,0 +1,217 @@
+import languageAssertions from '../../../../../test/unit/utils/languageAssertions';
+import { FormContent, FormFields, FormOptions, LanguageLookup } from '../../../../app/form/Form';
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.test.ts:2]: Please move this file under screening-questions. so the folder structure will be c100-rebuild/screening-questions

File: src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    alternativeRoutes: {
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts:173]: @kumarmadurai Please use prefix 'sq_' for the fields defined. This is part of screening questions

File: src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+      type: 'radios',
+      classes: 'govuk-radios',
+      values: [
+        {
+          label: l => l.one,
+          value: YesOrNo.YES,
+          subFields: {
+            agreementReason: {
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts:181]: Same as above, please prefix with 'sq_'

File: src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts (hunk 2 above)
Comment (yogendraU) [inline src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts:173]: please prefix it by sq_

File: src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+              type: 'textarea',
+              label: l => l.otherDetails,
+              labelSize: null,
+              validator: value => isFieldFilledIn(value) || isTextAreaValid(value),
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts:185]: @kumarmadurai These are optional question, please remove all validations & error messages added.

File: src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+            },
+          },
+        },
+        {
+          label: l => l.two,
+          value: YesOrNo.NO,
+        },
+      ],
+      validator: isFieldFilledIn,
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts:194]: Please remove the validation

File: src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts [hunk 6]
```diff
@@ -0,0 +1,211 @@
...
+  otherDetails: 'Provide details',
+  errors: {
+    agreementReason: {
+      required: 'Please provide details',
+    },
+  },
+});
+
+const cy = () => ({
+  title: 'Other ways to reach an agreement - walsh',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/alternative-resolution/alternative-routes/content.ts:90]: @kumarmadurai Please correct typo for **welsh**

File: src/main/steps/urls.ts [hunk 7]
```diff
@@ -243,6 +243,10 @@ export const C100_C1A_SAFETY_CONCERNS_OTHER_CONCERNS_DRUGS: PageLink = `${C100_C
 /** @C100 Document Submission */
 export const C100_DOCUMENT_SUBMISSION: PageLink = `${C100_URL}/document-submission`;
 
+/** @C100 Alternative Rsesolution */
+export const C100_ALTERNATIVE_RESOLUTION: PageLink = `${C100_URL}/alternative-resolution`;
```
Comment (yogendraU) [inline src/main/steps/urls.ts:247]: this should be screening-questions

Comment (channa-patil) [inline src/main/steps/urls.ts:247]: @kumarmadurai This is part of screening questions, we need to add these changes inside /screening-questions folder & in URL.

Comment (channa-patil) [reply src/main/steps/urls.ts:247]: @yogendraU Can you please add your inputs

**Review by channa-patil — CHANGES_REQUESTED** [review]

**Review by yogendraU — CHANGES_REQUESTED** [review]

File: src/main/steps/urls.ts (hunk 7 above)
Comment (yogendraU) [reply src/main/steps/urls.ts:247]: yes. it should be part of /screening-questions

**Pull request closed without merging**